examples/simple-store.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
```

Comment:
For example code (and other code used as documentation), it's helpful to put it into the public domain, so consumers can copy/paste it into their own projects without having to consider the licensing implications. This is supported by the [Mozilla license policy](https://www.mozilla.org/en-US/MPL/license-policy/), which says:

>Trivial bits of Mozilla Code, such as testcases or snippets of code used in documentation, should be put in the public domain in order to facilitate re-use. To do that, apply the Creative Commons Public Domain Dedication by using the following boilerplate:
>
>```
>Any copyright is dedicated to the Public Domain.
>http://creativecommons.org/publicdomain/zero/1.0/
>```

So let's put this example (and others we add) into the public domain.